source/deployment/cluster.rst

```diff
@@ -293,6 +293,14 @@ Mattermost runs periodic tasks via the `job server <https://docs.mattermost.com/
 
 :ref:`Run all job servers <command-line-tools-mattermost-jobserver>` with ``--noschedule flag``, then set ``JobSettings.RunScheduler`` to ``true`` in config.json for all app servers in the cluster. The cluster leader will then be responsible for scheduling recurring jobs.
 
+Plugins and High Availability
+^^^^^^^^^^^^^^^^
+
+As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically. 
+
+Note a slight behavior change in v5.15:
+When you install a plugin in 5.14 -> enable -> uninstall -> install the same plugin again, the plugin's initial state remained  enabled. As of v5.15, when you install a plugin -> enable -> uninstall -> install the same plugin again, the plugin's initial state will now be DISABLED.
```

Comment:
```suggestion
When you install a plugin in 5.14 via **enable > uninstall > install the same plugin again**, the plugin's initial state remained  enabled. As of 5.15, when you install a plugin via **enable > uninstall > install the same plugin again**, the plugin's initial state will now be disabled.
```

Reply:
This might need more details on where the "enable > uninstall > install the same plugin again" settings are.

Reply:
Does this seem awfully detailed? Should we rather say something like: `Plugins retained their previous state on reinstall. As of 5.15, plugins are always disabled on install.` Or something like that -- not a word wizard unfortunately :P

Reply:
Perhaps @justinegeffen can help us on her first day xD 👋

Reply:
I re-worded it to simplify.

Reply:
@justinegeffen Please, let me know if you have any suggestions.